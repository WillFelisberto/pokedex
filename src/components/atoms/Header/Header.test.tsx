import { render, screen } from '@/tests/test-utils';

import { Header } from '.';

describe('<Header />', () => {
  it('should render the header', () => {
    const { container } = render(<Header />);

    expect(screen.getByText(/Pokédex/i)).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });

  it('should render the header with class', () => {
    render(<Header />);

    const header = screen.getByText(/Pokédex/i);

    expect(header).toHaveClass(
      'text-lg sm:text-xl font-bold font-[family-name:var(--font-press-start-2p)]'
    );
  });
});
