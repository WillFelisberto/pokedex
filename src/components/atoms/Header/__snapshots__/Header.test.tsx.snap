// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the header 1`] = `
<div>
  <header
    class="bg-gray-100 place-content-center flex items-center justify-center w-full text-gray-500 p-4 sm:p-6"
  >
    <div
      class="max-w-[1440px] w-full flex-shrink-0 items-center gap-2"
    >
      <h1
        class="text-lg sm:text-xl font-bold font-[family-name:var(--font-press-start-2p)]"
      >
        Pok√©dex
      </h1>
    </div>
  </header>
</div>
`;
